backend/src/org/commcare/resources/ResourceManager.java

```diff
@@ -136,8 +136,17 @@ protected void loadProfileIntoTable(ResourceTable table,
             throws UnfullfilledRequirementsException,
```

Comment:
Instead of relying on name overloading, can you change this method's name to include the fact that a remote authority is being?